include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,508 @@
+    //VkBool32           multiviewGeometryShader;
+    //VkBool32           multiviewTessellationShader;
+    //VkBool32           variablePointersStorageBuffer;
+    //VkBool32           variablePointers;
+    //VkBool32           protectedMemory;
+    //VkBool32           samplerYcbcrConversion;
+    bool shaderDrawParameters = false;
+
+    /* Vulkan 1.2 Core */
+    //VkBool32           samplerMirrorClampToEdge;
+    bool drawIndirectCount = false;
+    //VkBool32           storageBuffer8BitAccess;
+    //VkBool32           uniformAndStorageBuffer8BitAccess;
+    //VkBool32           storagePushConstant8;
+    //VkBool32           shaderBufferInt64Atomics;
+    //VkBool32           shaderSharedInt64Atomics;
+    //VkBool32           shaderFloat16;
+    //VkBool32           shaderInt8;
+    //VkBool32           descriptorIndexing;
+    //VkBool32           shaderInputAttachmentArrayDynamicIndexing;
+    //VkBool32           shaderUniformTexelBufferArrayDynamicIndexing;
+    //VkBool32           shaderStorageTexelBufferArrayDynamicIndexing;
+    //VkBool32           shaderUniformBufferArrayNonUniformIndexing;
+    //VkBool32           shaderSampledImageArrayNonUniformIndexing;
+    //VkBool32           shaderStorageBufferArrayNonUniformIndexing;
+    //VkBool32           shaderStorageImageArrayNonUniformIndexing;
+    //VkBool32           shaderInputAttachmentArrayNonUniformIndexing;
+    //VkBool32           shaderUniformTexelBufferArrayNonUniformIndexing;
+    //VkBool32           shaderStorageTexelBufferArrayNonUniformIndexing;
+    //VkBool32           descriptorBindingUniformBufferUpdateAfterBind;
+    //VkBool32           descriptorBindingSampledImageUpdateAfterBind;
+    //VkBool32           descriptorBindingStorageImageUpdateAfterBind;
+    //VkBool32           descriptorBindingStorageBufferUpdateAfterBind;
+    //VkBool32           descriptorBindingUniformTexelBufferUpdateAfterBind;
+    //VkBool32           descriptorBindingStorageTexelBufferUpdateAfterBind;
+    //VkBool32           descriptorBindingUpdateUnusedWhilePending;
+    //VkBool32           descriptorBindingPartiallyBound;
+    //VkBool32           descriptorBindingVariableDescriptorCount;
+    //VkBool32           runtimeDescriptorArray;
+    //VkBool32           samplerFilterMinmax;
+    //VkBool32           scalarBlockLayout;
+    //VkBool32           imagelessFramebuffer;
+    //VkBool32           uniformBufferStandardLayout;
+    //VkBool32           shaderSubgroupExtendedTypes;
+    //VkBool32           separateDepthStencilLayouts;
+    //VkBool32           hostQueryReset;
+    //VkBool32           timelineSemaphore;
+    bool bufferDeviceAddress = false;
+    //VkBool32           bufferDeviceAddressCaptureReplay;
+    //VkBool32           bufferDeviceAddressMultiDevice;
+    //VkBool32           vulkanMemoryModel;
+    //VkBool32           vulkanMemoryModelDeviceScope;
+    //VkBool32           vulkanMemoryModelAvailabilityVisibilityChains;
```

Comment:
probably best to report `false` on GL and GLES

Reply:
So you want me to uncomment vulkanMemoryModel stuff? I don't even know what they are used for :D

Reply:
its about the implementation officially supporting the whole "synchronizes with, visible, available" shit